src/Sylius/Behat/Context/Api/Shop/ProductContext.php

```diff
@@ -83,6 +94,39 @@ public function iSearchForProductsWithName(string $name)
         $this->client->filter();
     }
 
+    /**
+     * @Then I should see :amount product reviews
+     */
+    public function iShouldSeeProductReviews(int $amount): void
```

Comment:
I would be for moving these three steps to a separate context `ProductReviewContext`

Reply:
I was thinking about it, but after all we are checking products details page, so decided to put it in product context.

Reply:
In API context, it is not checking the products details page, but products review index